person_capture/gui_app.py

```
@@ -808,6 +808,199 @@ def _prescan_face_conf_value(cfg) -> float:
+    def _cfg_bool(value, default: bool = False) -> bool:
+        if isinstance(value, bool):
+            return value
+        if isinstance(value, str):
+            v = value.strip().lower()
+            if v in {"1", "true", "yes", "on"}:
+                return True
+            if v in {"0", "false", "no", "off", ""}:
+                return False
+        try:
+            return bool(value)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)